### DISCUSSION TITLE
Feedback & Feature Request

### DISCUSSION THREAD

**Discussion by ph1p** [ph1p/figma-measure#14] [Ideas]
If you have feedback or feature suggestions, post them here.

**Comment by iqzap** [comment]
Measure the padding  :)

**Reply by ph1p** [reply]
I'm already at it (: you can see my progress in this branch: https://github.com/ph1p/figma-measure/blob/version2.2/src/main/index.ts#L930 
The only thing missing is a reasonable GUI, since I also implemented things like a multiplier for the measurement values.

**Reply by iqzap** [reply]
Thanks bro, can't wait for it 👍

**Comment by megaroeny** [comment]
Please, can you not lock child layers? Especially for folks wanting to move objects around, the plugin's refresh/sync button doesn't usually work. I opened an issue about this, thanks!

https://github.com/ph1p/figma-measure/issues/30

**Comment by markmanfrey** [comment]
Can you incorporate color and font style names into the Tooltip?

**Reply by markmanfrey** [reply]
I've managed to get the styles added to the tooltip on a local branch. Would you be interested taking a look and possibly folding it into plugin officially? I could make a branch/pull request and it'd be there if you wanted it.

**Reply by ph1p** [reply]
Super! Just create a PR, I'll look over it and there should be no problems to insert it :)

**Reply by ph1p** [reply]
I am currently preparing a new version, so it fits great

**Reply by markmanfrey** [reply]
Sweet. Before I make a PR, i'm trying to add a check to see if a style exists on the object. Do you happen to know the trick to that?

**Reply by ph1p** [reply]
I was so curious how to get all the data that I implemented it myself ^^. At the moment it looks like this. (not yet published)
![screen](https://user-images.githubusercontent.com/15351728/150206210-12d4e579-6463-42d9-9dc5-472f5b62b3b6.png)

The name will be displayed instead of the color. The best way for me is if you create a small image and show me how you want it.
Code: https://github.com/ph1p/figma-measure/blob/master/src/main/helper.ts#L133

**[UPDATE]**
![screen](https://user-images.githubusercontent.com/15351728/150219870-8249812d-e445-402f-8a64-80c911c23fd1.png)

**Reply by markmanfrey** [reply]
That's awesome! What you have totally works. For mine I just re-ordered the rect so it was last and have all the styles (if assigned) be first.

<img width="300" alt="Screen Shot 2022-01-19 at 9 09 39 PM" src="https://user-images.githubusercontent.com/79664563/150277351-67a39cf7-8c02-40d4-bd07-b34dd6850378.png">

**Reply by ph1p** [reply]
Published it today (:

**Reply by koltarmartin** [reply]
Hi. I see that some progress has been made on including text style names in the tooltip. I wanted to see when this will be added to the plugin. I still only see the font name in the tool tip. Thanks :)

**Reply by markmanfrey** [reply]
@ph1p how do we get the latest so we can see the style names in the tooltip?

**Reply by ph1p** [reply]
@markmanfrey today I have released a new version. I'm doing my best to work through all kinds of feature requests and bug fixes. Sorry that sometimes it takes a little while as I still have to do my daily job 😊

**Comment by koltarmartin** [comment]
Love this plugin. Component instance name and variant settings would be awesome too
![Screen Shot 2022-02-15 at 11 15 59 AM](https://user-images.githubusercontent.com/49528470/154114588-0f53de9d-5550-46f7-bb0f-575144cde14d.png)
.

**Reply by koltarmartin** [reply]
@ph1p The update to include variant settings is awesome! This is going to save me so much time. Thank you! Any chance we can get text style names too? Not a huge deal but would be nice :)

**Reply by markmanfrey** [reply]
Oh i got confused. I thought the text style names were added. I'd love that addition too.

**Comment by kemie** [comment]
This plugin is awesome! Two requests:

- Option to include  effects (effect styles too) on callout/tooltip
- Be able to configure text size with formulas the same way you can configure measurements (really useful for displaying font size in ems/rems)

**Reply by ph1p** [reply]
First, thank you :) Sorry it took so long, today I added effect informations to the tooltip and I'm currently looking for a good way to implement the font size formula.

**Reply by kemie** [reply]
Thank you so much for the great work!

**Comment by maciej-panecki-wttech** [comment]
Can you add option to include token name instead of value if it is defined?
